app/employee-app/backend/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "wso2"
+name = "employee_app"
+version = "1.0.0"
+distribution = "2201.8.6"
```

Comment:
Can we update this to the latest minor version of Ballerina Update 8? (2201.8.8) [1]

[1] - https://ballerina.io/downloads/archived/#2201.8.8

Reply:
https://github.com/wso2/people-ops-suite/pull/1#discussion_r1818935526